client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -367,6 +350,72 @@ export default class StartInstanceTool extends PureComponent {
     });
   }
 
+  handleStartError(tab, error) {
+    const {
+      log,
+      displayNotification
+    } = this.props;
+
+    displayNotification({
+      type: 'error',
+      title: START_INSTANCE_FAILED,
+      content: 'See the log for further details.',
+      duration: 10000
+    });
+
+    log({
+      category: 'start-instance-error',
+      message: error.problems || error.message
+    });
+  }
+
+  handleDeploymentSuccess(tab, deployment) {
+
+    const {
+      triggerAction
+    } = this.props;
+
+    // notify interested parties
+    triggerAction('emit-event', {
+      type: 'deployment.done',
+      payload: {
+        deployment,
+        isRun: true
```

Comment:
I still do not understand why we don't just add a _run_ event, too. Would allow us to separate deploy only from run (`#deploy` - `#run`) and provide details in actual run usage (and results), too.

Reply:
I added an issue for this https://github.com/camunda/camunda-modeler/issues/1993